examples/camera-app/camera-common/src/camera-app.cpp

```diff
@@ -21,16 +21,60 @@ using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::Chime;
+using namespace chip::app::Clusters::WebRTCTransportProvider;
+using namespace chip::app::Clusters::CameraAvStreamManagement;
 
 template <typename T>
 using List   = chip::app::DataModel::List<T>;
 using Status = Protocols::InteractionModel::Status;
 
+CameraApp::CameraApp(chip::EndpointId aClustersEndpoint, CameraDeviceInterface * aCameraDevice)
+{
+    mEndpoint     = aClustersEndpoint;
```

Comment:
if we only use mEndpoint transitory within this function, why save it to a class member?